### DISCUSSION TITLE
Help for a beginner

### DISCUSSION THREAD

**Discussion by selvaprakash** [deforum-art/sd-webui-text2video#190] [Q&A] [answered]
Hi All,

I'm getting into AI projects now. I need help in running this model as such. I'm able to run the model from huggingface. But this version, I believe is upgraded. 
When I try to run the text2vid.py I get an error  saying 'modules' does not exist (lines 22,23). And I don't see any such folder called modules in the repo. Am I missing something? 

Any help would be greatly appreciated.

Thanks.

**Comment by selvaprakash** [comment, marked as answer]
Hi All, 
I was able to figure it out. I didn't know that I need to have Stable Diffusion first for this extension to work. That was very dumb of me 🤦‍♂️ . It worked well after that. So, for anyone starting this without knowing about stable diffusion etc. Follow the below process.
1) Clone the Stable diffusion repo from https://github.com/selvaprakash/stable-diffusion-webui
2) Go to the extensions folder in that repo and clone this extension
3) Go to models folder and create these two subfolders one below the other ModelScope ->t2v.
4) Copy the weights and config files from Huggingface repository of Modelscope https://huggingface.co/damo-vilab/modelscope-damo-text-to-video-synthesis/tree/main
You should be good to to. Hope this helps.

**Reply by B34STW4RS** [reply]
Did you just plug your own weird repo, as an answer to your own question? Why would anyone do any of this when you can get the official RC and just tick the extension?
